pkgs/tools/video/tsduck/default.nix

```
@@ -66,16 +66,22 @@ stdenv.mkDerivation rec {
     "NOPCSC=1"
     "NORIST=1"
     "NOVATEK=1"
-  ] ++ installFlags;
+  ] ++ installFlags ++ extraFlags;
```

Using `lib.optionals` instead of `if … then … else` allows to inline conditional flags conveniently:
```suggestion
  ] ++ lib.optionals stdenv.isDarwin [
    "NOTEST=1"
    "CXXFLAGS_EXTRA=-Wno-unsafe-buffer-usage"
  ] ++ installFlags;
```